content/projects/data-science-specific/data-visualisation/mobile-money-viz/_index.md

```diff
@@ -23,11 +23,11 @@ tags:
 title: Financial Services Use in Tanzania
 ---
 
-## Background
+## About the DataSet
 
 The [training](training.csv) dataset contains demographic information and what financial services are used by approximately 10,000 individuals across Tanzania. This data was extracted from the FSDT Finscope 2017 survey and prepared specifically for this challenge.
 
-Each individual is classified into four mutually exclusive categories:
+### Each individual is classified into four mutually exclusive categories:
```

Comment:
This seems like a perfectly normal sentence.  Why is it a heading?

Choose one of these:

```
Each individual is classified into four mutually exclusive categories:

...
```

or

```
### Categories

Each individual is classified into four mutually exclusive categories:

...
```